src/AutoUI/index.tsx

```diff
@@ -144,7 +142,7 @@ export interface AutoUIProps<T> extends Omit<Material.BoxProps, 'onChange'> {
 		} | null;
 	}) => void;
 	/** Information from a server side pagination */
-	pagination?: Pagination;
+	pagination?: Pagination | undefined;
```

Comment:
We need both the `?` and the `undefined`?

Reply:
Also confused why the pagination typing in `Lenses/types/index` is purely `pagination: Pagination` but here we need/accept an `undefined`

Reply:
> Also confused why the pagination typing in Lenses/types/index is purely pagination: Pagination but here we need/accept an undefined

Ah I see because once we pass it to the lens it's guaranteed to exist via `internalPagination`